postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -117,6 +117,10 @@ instances:
 #    collect_default_database: False
 #
 
+#    Tag metrics and checks with `replication_role:<master|standby>`, default to false
```

Comment:
Any specific reason to put this behind a feature flag/default to false? Doesn't necessarily need to change, just curious of the decision.

Reply:
I just didn't want to add the tag by default in case the customers aren't even using replication to begin with. Also, for existing users, I just didn't want to suddenly add a tag to the metrics unless they need/want it. Save on (probably neglible) data size/ storage as well.